__tests__/app.test.js

```diff
@@ -148,9 +148,53 @@ describe("app", () => {
         .get("/api/reviews/999")
         .expect(404)
         .then(({ body }) => {
-          console.log(body.msg);
           expect(body.msg).toBe("Not Found");
         });
     });
   });
 });
+describe("/api/reviews/:review_id/comments", () => {
+  it("200: GET: it should respond with an object of comments for the given review_id", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Object);
+      });
+  });
+  it("200:GET: should respond with an empty array if review_id has no comments", () => {
+    const reviewId = 100;
+    return request(app)
+      .get(`/api/reviews/${reviewId}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
+        if (body.comments.length === 0) {
+          expect(body).toEqual({ comments: [] });
+        }
+      });
+  });
+  it("200:GET: should respond with the correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
```

Comment:
review_id 1 doesnt actually have any comments, which is sending back an empty array and because the array is empty this forEach is never being invoked, so this tests isn't running